src/backend/language.cpp

```diff
@@ -79,238 +80,1049 @@ LanguageMap::LanguageMap() {
+
+    addLanguage({"sa"}, QT_TRANSLATE_NOOP("QObject", "Sanskrit"));
+    addLanguage({"sab"}, QT_TRANSLATE_NOOP("QObject", "Buglere"));
+    addLanguage({"sah-Cyrl"}, QT_TRANSLATE_NOOP("QObject", "Yakut (Cyrillic script)"));
+    addLanguage({"sbe"}, QT_TRANSLATE_NOOP("QObject", "Saliba"));
     //  addLanguage({"sc"}, QT_TRANSLATE_NOOP("QObject", "Sardinian"));
     addLanguage({"sco"}, QT_TRANSLATE_NOOP("QObject", "Scots"));
     //  addLanguage({"sd"}, QT_TRANSLATE_NOOP("QObject", "Sindhi"));
     //  addLanguage({"se"}, QT_TRANSLATE_NOOP("QObject", "Northern Sami"));
+    addLanguage({"seh"}, QT_TRANSLATE_NOOP("QObject", "Sena"));
+    addLanguage({"sey"}, QT_TRANSLATE_NOOP("QObject", "Secoya"));
     //  addLanguage({"sg"}, QT_TRANSLATE_NOOP("QObject", "Sango"));
+    addLanguage({"sgb"}, QT_TRANSLATE_NOOP("QObject", "Mag-Anchi Ayta"));
+    addLanguage({"sgz"}, QT_TRANSLATE_NOOP("QObject", "Sursurunga"));
+    addLanguage({"shj"}, QT_TRANSLATE_NOOP("QObject", "Shatt"));
+    addLanguage({"shp"}, QT_TRANSLATE_NOOP("QObject", "Shipibo-Conibo"));
     //  addLanguage({"si"}, QT_TRANSLATE_NOOP("QObject", "Sinhalese"));
+    addLanguage({"sim"}, QT_TRANSLATE_NOOP("QObject", "Mende (Papua New Guinea)"));
+    addLanguage({"sja"}, QT_TRANSLATE_NOOP("QObject", "Epena"));
     addLanguage({"sk"}, QT_TRANSLATE_NOOP("QObject", "Slovak"));
     addLanguage({"sl"}, QT_TRANSLATE_NOOP("QObject", "Slovenian"));
+    addLanguage({"sll"}, QT_TRANSLATE_NOOP("QObject", "Salt-Yui"));
     //  addLanguage({"sm"}, QT_TRANSLATE_NOOP("QObject", "Samoan"));
-    //  addLanguage({"sn"}, QT_TRANSLATE_NOOP("QObject", "Shona"));
+    addLanguage({"smk"}, QT_TRANSLATE_NOOP("QObject", "Bolinao"));
+    addLanguage({"sml"}, QT_TRANSLATE_NOOP("QObject", "Central Sama"));
+    addLanguage({"sn"}, QT_TRANSLATE_NOOP("QObject", "Shishona"));
+    addLanguage({"snc"}, QT_TRANSLATE_NOOP("QObject", "Sinaugoro"));
+    addLanguage({"snn"}, QT_TRANSLATE_NOOP("QObject", "Siona"));
+    addLanguage({"snp"}, QT_TRANSLATE_NOOP("QObject", "Siane"));
+    addLanguage({"sny"}, QT_TRANSLATE_NOOP("QObject", "Saniyo-Hiyewe"));
     addLanguage({"so"}, QT_TRANSLATE_NOOP("QObject", "Somali"));
+    addLanguage({"soq"}, QT_TRANSLATE_NOOP("QObject", "Kanasi"));
+    addLanguage({"soy"}, QT_TRANSLATE_NOOP("QObject", "Miyobe"));
+    addLanguage({"spa", "es"}, QT_TRANSLATE_NOOP("QObject", "Spanish"));
+    addLanguage({"spl"}, QT_TRANSLATE_NOOP("QObject", "Selepet"));
+    addLanguage({"spm"}, QT_TRANSLATE_NOOP("QObject", "Sepen"));
+    addLanguage({"spp"}, QT_TRANSLATE_NOOP("QObject", "Supyire Senoufo"));
+    addLanguage({"sps"}, QT_TRANSLATE_NOOP("QObject", "Saposa"));
+    addLanguage({"spy"}, QT_TRANSLATE_NOOP("QObject", "Sabaot"));
     addLanguage({"sq"}, QT_TRANSLATE_NOOP("QObject", "Albanian"));
-    //  addLanguage({"sr"}, QT_TRANSLATE_NOOP("QObject", "Serbian"));
+    addLanguage({"sr"}, QT_TRANSLATE_NOOP("QObject", "Srpski"));
```

Comment:
According to https://iso639-3.sil.org/code/srp this is Serbian, and also needs a "srp" abbreviation.